buildroot-external/board/pc/ova/kernel.config

```diff
@@ -70,19 +70,6 @@ CONFIG_SND_XEN_FRONTEND=y
 CONFIG_XEN_PVCALLS_FRONTEND=m
 CONFIG_XEN_WDT=y
 
-CONFIG_WLAN_VENDOR_INTEL=y
-CONFIG_IWL4965=m
-CONFIG_IWL3945=m
-CONFIG_IWLWIFI=m
-CONFIG_IWLDVM=m
-CONFIG_IWLMVM=m
-
-CONFIG_WLAN_VENDOR_REALTEK=y
-CONFIG_RTL_CARDS=m
-CONFIG_RTL8723BE=y
-
-CONFIG_BCMA=m
```

Comment:
I think this driver can be used on non PCIe targets as well. Not sure if that is actually the case though.